src/App.tsx

```diff
@@ -1,24 +1,69 @@
 import React from 'react';
+import classNames from 'classnames';
 import './App.scss';
 
-const goodsFromServer: string[] = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+class App extends React.Component {
+  state = {
+    goodsFromServer: [
+      'Dumplings',
+      'Carrot',
+      'Eggs',
+      'Ice cream',
+      'Apple',
+      'Bread',
+      'Fish',
+      'Honey',
+      'Jam',
+      'Garlic',
+    ],
```

Comment:
add goodsFromServer variable outside of the component, as it was done initially 

```suggestion
    goods: goodsFromServer,
```